demos/polls/setup.cfg

```diff
@@ -0,0 +1,17 @@
+[tool:pytest]
```

Comment:
FWIW, my preference is to use a separate `pytest.ini` file because it's easier to find where it's configured, and it's easier to sync individual configs across projects.

Reply:
Alright, happy to move it, I was just copying from the aiohttp repo.

Reply:
No problem, just sharing.

Reply:
Here's a larger example, by the way: https://github.com/cherrypy/cheroot/blob/main/pytest.ini